server/src/test/java/org/opensearch/indices/replication/SegmentReplicationTargetServiceTests.java

```diff
@@ -54,6 +60,14 @@ public void setUp() throws Exception {
         when(replicationSourceFactory.get(indexShard)).thenReturn(replicationSource);
 
         sut = new SegmentReplicationTargetService(threadPool, recoverySettings, transportService, replicationSourceFactory);
+        cp = indexShard.getLatestReplicationCheckpoint();
+        newCheckpoint = new ReplicationCheckpoint(
```

Comment:
Nitpick - more descriptive variable names, please. `cp` is hard to decipher at first glance and `newCheckpoint` is confusing when it's being initialized to be equal to `cp`